src/modules/solution/type/multiple_choice_solution.py

```diff
@@ -0,0 +1,12 @@
+from .abstract_solution import AbstractSolution
+
+
+class MultipleChoiceSolution(AbstractSolution):
+    def __init__(self, solution: list[str]):
```

Comment:
It's cleaner to pass in the actual solution instead of the index of the options.